core/src/main/java/org/openscience/jchempaint/action/CopyPasteAction.java

```diff
@@ -333,17 +332,13 @@ else if(renderModel.getHighlightedBond()!=null){
                         toPaste = sp.parseSmiles(
                                 ((String) transfer.getTransferData(
                                         DataFlavor.stringFlavor)).trim());
-                        toPaste = new FixBondOrdersTool().kekuliseAromaticRings(toPaste);
+                        Kekulization.kekulize(toPaste);
```

Comment:
You should remove this, it's a no-op as the SmilesParser will provide 'sane' Kekulized SMILES (unless it's turned off).